modules/ossm-control-plane-templates.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/prepare-to-deploy-applications-ossm.adoc
+
+[id="ossm-control-plane-templates_{context}"]
+= Creating control plane templates
+
+When you configure control plane templates, which follow the same syntax as the `ServiceMeshControlPlane`, users inherit settings in a hierarchical factor. The Operator is delivered with a `default` template with default settings for {ProductName}. To add custom templates you must create a ConfigMap named `smcp-templates` in the `openshift-operators` and mount the ConfigMap in the Operator at `/usr/local/share/istio-operator/templates`.
```

Comment:
Control planes are a new concept here as far as I can tell. Given that, is there a better way to start than "when you configure control plane templates"? 

This seems to be a couple of ideas mixed together.
* You can take a servicemesh control plane that you've created and tested and turn it into a template by including it in the `smcp-templates` configmap. These templates can build off of one another inheriting (accounting team -> default template for Service Mesh -> common settings between Maistra and Service Mesh)
* The operator ships with a temlate named default. If an organization chooses to, they can override the default template in the configmap, which means that all templates will build off of this.

Reply:
Sounds like there should be a section describing what templates are and how/why they are used, followed up by a blurb describing how to create a custom template, followed by the blurb on how to configure the operator to use the custom template.

Reply:
Can we add namespace after this statement?
"... ConfigMap named `smcp-templates` in the `openshift-operators`..."

So it reads:
"... ConfigMap named `smcp-templates` in the `openshift-operators` namespace..."

Reply:
This module has been changed significantly. Please review the updates.